components/tests/test_component_marching.cpp

```diff
@@ -167,25 +190,32 @@ TEST_CASE("marching_component_tri", "[components][marching]")
         }
     }
 
-    if (false) {
-        wmtk::io::ParaviewWriter
-            writer("marching_2d_result", "vertices", m, true, false, true, false);
+    if (true) {
+        wmtk::io::ParaviewWriter writer(
+            "/home/zhouyuan/workplace/toolkit/wildmeshing-toolkit/data/marching_2d_result",
```

Comment:
No absolute paths please.
Also this should not print anything by default --> `if(false)`